service/raft/raft_rpc.cc

```diff
@@ -77,7 +77,7 @@ raft_rpc::two_way_rpc(sloc loc, raft::server_id id,
     }
     return verb(&_messaging, netw::msg_addr(*ip_addr), db::no_timeout, _group_id, _my_id, id, std::forward<Args>(args)...)
         .handle_exception_type([loc= std::move(loc), id] (const seastar::rpc::closed_error& e) {;
-            const auto msg = fmt::format("Failed to execute {} on leader {}: {}", loc.function_name(), id, e);
+            const auto msg = fmt::format("Failed to execute {}, destination {}: {}", loc.function_name(), id, e);
```

Comment:
shouldn't this be `on destination`?

Reply:
I think that wouldn't be very accurate. The function that fails to execute is happening here, on the sender. It might fail because of some problem on the destination OR because of some problem on the sender, we don't know. "on destination" would suggest that the failure always happens on the destination.

Reply:
Ok, got it.